modules/installation-azure-permissions.adoc

```diff
@@ -6,10 +6,11 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-Your Microsoft Azure account must have the following roles for the subscription
+You can use your Microsoft Azure account for `az login` and browsing in the web console. The Microsoft Azure account has the correct permissions to create the service principal account. The Microsoft Azure account uses the created service principal account to run the installer and create the cluster. Your Microsoft Azure account must have the following roles for the subscription
```

Comment:
How about changing "The Microsoft Azure account uses the created service principal account to run the installer and create the cluster" to "The created service principal account is used to run the installer and create the cluster" ?

Reply:
acked